nengo/networks/workingmemory.py

```diff
@@ -31,22 +30,20 @@ def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
         nengo.Connection(net.mem.output, net.diff.input, transform=-1)
 
         # feed difference into integrator
-        with mem_config:
-            nengo.Connection(net.diff.output, net.mem.input,
-                             transform=difference_gain)
+        nengo.Connection(net.diff.output, net.mem.input,
+                         transform=difference_gain,
+                         synapse=nengo.Lowpass(difference_synapse))
```

Comment:
It happens [here in the `SynapseParam`](https://github.com/nengo/nengo/blob/master/nengo/synapses.py#L272). We mostly keep around the ability to pass in a number for convenience and backwards compatibility.